docs/howto/openstack/nova/new-server.md

```diff
@@ -0,0 +1,348 @@
+	```
+		
+	...and then go ahead and create it:
+			
+	```bash
+	openstack server create \
+		--flavor b.1c1gb \
+		--image $IMAGE_NAME \
+		--boot-from-volume 20 \
+		--network nordostbahnhof \
+		--security-group default \
+		--key-name bahnhof \
+		--wait \
+		zug
+	```
+		
+	(For clarity's sake, and with the exception of `IMAGE_NAME`, we 
+	used the actual values and not the variables we so meticulously set.) 
+	The `--wait` parameter is optional. Whenever you choose to 
+	use it, you get back control of your terminal only after the server is 
+	readily available in {{extra.brand}} Cloud.
+	
+	To connect to your server remotely, you need to create a floating IP
+	for the external network in the {{extra.brand}} Cloud, and then assign
+	this IP to your server. First, create the floating IP: 
+	
+	```bash
+	openstack floating ip create ext-net
+	```
+		
+	See all floating IPs...
+		
+	```bash
+	openstack floating ip list
+	```
+		
+	...and assign the one you just created to your server:
+		
+	```bash
+	openstack server add floating ip zug 198.51.100.12
+	```
+		
+	The username of the default user account in the Ubuntu image is 
+	`ubuntu`, so now you can connect to your remote server via SSH without 
+	typing a password:
+	
+	```bash
+	ssh ubuntu@198.51.100.12
+	```
+
+## Viewing information about the newly created server
+=== "{{extra.gui}}"
+	From the {{extra.brand}} Cloud Control Panel you may, at any 
+	time, see all servers and get detailed information regarding each one 
+	of them; expand the left-hand side vertical pane, click _Compute_, then 
+	_Servers_, and, in the central pane, select the region you want. 
+		
+	![Servers and details](assets/new-server/shot-09.png)
+=== "OpenStack CLI"
+	To see all available servers in the region, type:
+		
+	```bash
+	openstack server list
+	```
+		
+	You can always get specific information on a particular server:
+	
+	```bash
+	openstack server show zug
+	```
+## Connecting to the server console
+=== "{{extra.gui}}"		
+	While viewing information regarding your server, you may get 
+	its public IP  address (e.g., from the _Addresses_ tab) and connect to 
+	it remotely. Alternatively, you may launch a web console and log in; 
+	click on the three-dot icon on the right of the server header, and from 
+	the pop-up menu that appears select _Remote Console_.  
+		
+	![Launch remote console](assets/new-server/shot-10.png)
+		
+	A new window pops up, and that's your web console to your 
+	{{extra.brand}} Cloud server. Please note that this window cannot be 
+	resized but can be opened on a new browser window or tab.
+		
+	![Server console](assets/new-server/shot-11.png)
+=== "OpenStack CLI"
+	You may have access to the web console of your server, and you need the 
+	corresponding URL for it:
+	
+	```bash
+	openstack console url show zug
+	```
+	
+	Usage of the web console is discouraged, though. Instead, securely connecting 
+	via SSH without typing a password is strongly encouraged.
```

Comment:
The most recent change creates a bit of an ugly repetition, and also we don't need to use passive voice twice, here. Instead, let's use:

```markdown
Using the web console is discouraged, though. Instead, securely connect to 
to your server via SSH.
```

Reply:
Done!